src/python/oftest/testutils.py

```diff
@@ -499,6 +499,87 @@ def qinq_tcp_packet(pktlen=100,
 
     return pkt
 
+def simple_vxlan_packet(pktlen=300,
+                        eth_dst='00:01:02:03:04:05',
+                        eth_src='00:06:07:08:09:0a',
+                        dl_vlan_enable=False,
+                        vlan_vid=0,
+                        vlan_pcp=0,
+                        dl_vlan_cfi=0,
+                        ip_src='192.168.0.1',
+                        ip_dst='192.168.0.2',
+                        ip_tos=0,
+                        ip_ttl=64,
+                        ip_id=1,
+                        udp_sport=1234,
+                        udp_dport=4789,
+                        with_udp_chksum=True,
+                        ip_ihl=None,
+                        ip_options=False,
+                        vxlan_reserved1=0x000000,
+                        vxlan_vni = 0xaba,
+                        vxlan_reserved2=0x00,
+                        inner_frame = None):
+    """
+    Return a simple dataplane VXLAN packet
+    Supports a few parameters:
+    @param len Length of packet in bytes w/o CRC
+    @param eth_dst Destination MAC
+    @param eth_src Source MAC
+    @param dl_vlan_enable True if the packet is with vlan, False otherwise
+    @param vlan_vid VLAN ID
+    @param vlan_pcp VLAN priority
+    @param ip_src IP source
+    @param ip_dst IP destination
+    @param ip_tos IP ToS
+    @param ip_ttl IP TTL
+    @param ip_id IP Identification
+    @param udp_sport UDP source port
+    @param udp_dport UDP dest port (IANA) = 4789 (VxLAN)
+    @param vxlan_reserved1 reserved field (3B)
+    @param vxlan_vni VXLAN Network Identifier
+    @param vxlan_reserved2 reserved field (1B)
+    @param inner_frame The inner Ethernet frame
```

Comment:
looks like we're missing comments for a few parameters (such as "with_udp_chksum") and some param names have changed (such as "pktlen" instead of "len").  can you double-check?  thanks.